lib/espree.js

```diff
@@ -109,7 +110,7 @@ export default () => Parser => {
                 allowReturnOutsideFunction: options.allowReturnOutsideFunction,
 
                 // Collect tokens
-                onToken: token => {
+                onToken(token) {
```

Comment:
This is because in the new `eslint-config-eslint` the option [`avoidExplicitReturnArrows: true`](https://eslint.org/docs/latest/rules/object-shorthand#avoidexplicitreturnarrows) is set.